src/backend-gdix.c

```diff
@@ -113,6 +113,8 @@ gdix_init(void)
     GPA(SetPixelOffsetMode, (dummy_GpGraphics*, dummy_GpPixelOffsetMode));
     GPA(SetSmoothingMode, (dummy_GpGraphics*, dummy_GpSmoothingMode));
     GPA(TranslateWorldTransform, (dummy_GpGraphics*, float, float, dummy_GpMatrixOrder));
+    GPA(SetWorldTransform, (dummy_GpGraphics*, dummy_GpMatrix*, dummy_GpMatrixOrder));
+    GPA(CreateMatrix2, (float, float, float, float, float, float, dummy_GpMatrix**));
```

Comment:
I guess that if you need `GdipCreateMatrix2()`, you should likely also need `GdipDeleteMatrix()`.

Reply:
Forgot that the initialized matrix needs to get deleted.